src/ripple/basics/impl/contract.cpp

```diff
@@ -17,17 +17,36 @@
 */
 //==============================================================================
 
-#ifndef RIPPLE_LEDGER_VIEWAPIBASICS_H_INCLUDED
-#define RIPPLE_LEDGER_VIEWAPIBASICS_H_INCLUDED
+#include <BeastConfig.h>
+#include <ripple/basics/contract.h>
 
 namespace ripple {
 
-enum FreezeHandling
+namespace detail {
+
+void
+accessViolation()
 {
-    fhIGNORE_FREEZE,
-    fhZERO_IF_FROZEN
-};
+    // dereference memory
+    // location zero
+    int* j = 0;
+    *j++;
```

Comment:
You can ignore this for now - I'll be fixing it in an upcoming commit to avoid the dereference completely and using std functionality.